packages/web/src/services/AudiusBackend.ts

```diff
@@ -538,7 +603,7 @@ class AudiusBackend {
           error: false,
           web3Config: await AudiusLibs.configExternalWeb3(
             REGISTRY_ADDRESS,
-            web3.currentProvider,
+            window.web3.currentProvider,
```

Comment:
seems like a potential breaking change here, so wanted to flag in case

Reply:
Good flag, double checked that `window.web3` and `web3` are the same (injected by metamask). We are also null checking `window.web3` above so this seems more correct